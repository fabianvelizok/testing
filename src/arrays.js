export const fruits = ['apple', 'orange', 'strawberry'];
