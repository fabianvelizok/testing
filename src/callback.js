export const callback = cb => cb && cb('It a callback');
