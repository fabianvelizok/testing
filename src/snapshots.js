export const lastNames = ['Darell', 'Watson'];

export const person = {
  gender: 'Male',
  id: 1,
  lastName: 'Veliz',
  name: 'Fabian',
};

export const randomPerson = {
  gender: 'Female',
  id: Math.floor(Math.random() * 100),
  lastName: lastNames[Math.floor(Math.random() * 2)],
  name: 'Mary'
};
