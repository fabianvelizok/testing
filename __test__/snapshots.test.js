import { person, randomPerson } from '../src/snapshots';

describe('Snapshots', () => {
  it('Person snapshot should not change', () => {
    expect(person).toMatchSnapshot();
  })

  it('RandomPerson snapshot should change', () => {
    expect(randomPerson).toMatchSnapshot({
      id: expect.any(Number),
      lastName: expect.any(String)
    });
  })
});
