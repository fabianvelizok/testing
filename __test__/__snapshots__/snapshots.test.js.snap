// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Person snapshot should not change 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "lastName": "Veliz",
  "name": "Fabian",
}
`;

exports[`Snapshots RandomPerson snapshot should change 1`] = `
Object {
  "gender": "Female",
  "id": Any<Number>,
  "lastName": Any<String>,
  "name": "Mary",
}
`;
